bazel/google_cloud_cpp_deps.bzl

```diff
@@ -16,270 +16,8 @@
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+load("//bazel:workspace0.bzl", "gl_cpp_workspace0")
+load("//bazel:development0.bzl", "gl_cpp_development0")
 
-def google_cloud_cpp_development_deps(name = None):
```

Comment:
It looks like this will certainly break `external/googleapis/renovate.sh`.

And you might want to check `bazel/deps-cache.py` too.

Reply:
Fixed both.